app/src/main/java/com/example/trempel/di/AppComponent.kt

```diff
@@ -16,5 +18,7 @@ internal interface AppComponent {
     }
 
     fun inject(pdpFragment: PdpFragment)
+    fun inject(mensCategoryFragment: MensCategoryFragment)
+    fun inject(categoryFragment: CategoryFragment)
```

Comment:
We shouldn't have separate fragments for all products and for men's products

Reply:
done